torch/_export/serde/serialize.py

```diff
@@ -189,15 +189,15 @@ def serialize_tensor_meta(t: torch.Tensor) -> TensorMeta:
 
 def serialize_call_spec(call_spec: ep.CallSpec) -> CallSpec:
     return CallSpec(
-        in_spec=pytree_to_str(call_spec.in_spec) if call_spec.in_spec else "",
-        out_spec=pytree_to_str(call_spec.out_spec) if call_spec.out_spec else "",
+        in_spec=treespec_dumps(call_spec.in_spec) if call_spec.in_spec else "",
+        out_spec=treespec_dumps(call_spec.out_spec) if call_spec.out_spec else "",
```

Comment:
I wonder if export should manually specify the protocol is 1. Since export is a prototype I might just avoid thinking about this for now

Reply:
Yeah, it probably should. The version of the ExportedModel is somewhere else but is morally the source of truth of the "version of the ExportedModel". Every ExportedModel version should be tied to a specific treespec serialization protocol